config/application.rb

```diff
@@ -20,5 +20,6 @@ class Application < Rails::Application
     # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
+    config.autoload_paths += ['lib']
```

Comment:
@Fryguy This is needed so we can constantize the handlers [here](https://github.com/ManageIQ/miq_bot/pull/59/files#diff-be7efa9d2bb56731b17b2413e503b575R6)